index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body class="background">
+    <header class="navbar">
+        <div class="logo">
+          <a href="#">MyLogo</a>
+        </div>
+        <div>
+          <a href="#">
+            <i class="fa fa-bars"></i>
+          </a>
+        </div>
+    </header>
+    <section class="onscreen">
+      <h1 class="bold">I'm Amanulla </h1>
+      <h1 class="bold">Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
Please stick to Figma design:

Your:
<img width="319" alt="image" src="https://user-images.githubusercontent.com/7243132/170593494-75fc5daf-0635-49c8-b2c3-ad1091da866a.png">

Figma
<img width="319" alt="image" src="https://user-images.githubusercontent.com/7243132/170593518-eb7fe707-5ee1-4501-ba79-f79119137c49.png">